test/SdkTests/SdkTests.csproj

```diff
@@ -46,8 +46,8 @@
     <Reference Include="BouncyCastle.Crypto, Version=1.8.9.0, Culture=neutral, PublicKeyToken=0e99375e54769942">
       <HintPath>..\packages\BouncyCastle.1.8.9\lib\BouncyCastle.Crypto.dll</HintPath>
     </Reference>
-    <Reference Include="DocuSign.eSign">
-      <HintPath>..\..\sdk\src\DocuSign.eSign\bin\Debug\net452\DocuSign.eSign.dll</HintPath>
+    <Reference Include="DocuSign.eSign, Version=5.6.2.0, Culture=neutral, PublicKeyToken=7fca6fcbbc219ede, processorArchitecture=MSIL">
```

Comment:
These should probably be reverted too, if anyone wants to run the tests they're going to need to point to their locally built `.dll` anyway rather than a specific version of NuGet package.

Reply:
Done